### PR TITLE
get raster as numpy array via wcs

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Auf Raster mit WCS zugreifen](https://trello.com/c/P34mfEge/232-auf-raster-mit-wcs-zugreifen)

File: geoengine/types.py [hunk 1]
```diff
@@ -44,6 +44,24 @@ def __init__(self, spatial_bounds: Tuple[float, float, float, float], time_inter
...
+            return [bbox[1], bbox[0], bbox[3], bbox[2]]
+        else:
+            return bbox
+
+    @property
+    def resolution_ogc(self) -> Tuple[float, float]:
+        # TODO: properly handle axis order
+        res = self.__resolution
+        if self.__srs == "EPSG:4326":
+            return [-res[1], res[0]]
```
Comment (jdroenner) [inline geoengine/types.py:61]: why does the sign change?

File: geoengine/workflow.py [hunk 2]
```diff
@@ -232,6 +235,28 @@ def plot_chart(self, bbox: QueryRectangle) -> VegaLite:
 
         return VegaLite(vega_spec)
 
+    def get_array(self, bbox: QueryRectangle) -> np.ndarray:
+        '''
+        Query a workflow and return the raster result as a numpy array
+        '''
+        session = get_session()
+
+        # TODO: properly build CRS string for bbox
```
Comment (jdroenner) [inline geoengine/workflow.py:244]: what is missing here?

File: geoengine/types.py (hunk 1 above)
Comment (michaelmattig) [reply geoengine/types.py:61]: afaik resolution is always positive and here I turn it into pixel step sizes which are negative in y direction for wgs84

File: geoengine/workflow.py (hunk 2 above)
Comment (michaelmattig) [reply geoengine/workflow.py:244]: really understand the schema and be able to output non-epsg urn strings

Comment (jdroenner) [reply geoengine/workflow.py:244]: mh i guess we would just replace epsg with sr-org?

File: geoengine/types.py (hunk 1 above)
Comment (jdroenner) [reply geoengine/types.py:61]: ok maybe we need to document that somewhere

File: geoengine/workflow.py (hunk 2 above)
Comment (michaelmattig) [reply geoengine/workflow.py:244]: yeah but there are other ogc specific formats, e.g. one for WGS84 with other axis order etc. I just wrote this todo to make clear that this is not the final way in which we build that string

**Pull request merged**